build.sbt

```diff
@@ -1,20 +1,16 @@
 import sbt.Keys._
 
-// import scalariform.formatter.preferences._
-
-// addCommandAlias("format-code",    ";compile:scalariformFormat;test:scalariformFormat;it:scalariformFormat")
-
-lazy val buildSettings = Seq(
-  organization       := "com.mediative",
-  scalaVersion       := "2.10.4",
-  crossScalaVersions := Seq("2.10.4", "2.10.5", "2.11.7"),
-  licenses += ("Apache-2.0", url("http://www.opensource.org/licenses/apache2.0"))
-)
+import scalariform.formatter.preferences._
 
 enablePlugins(GitVersioning)
 git.useGitDescribe := true
 
-lazy val commonSettings = Seq(
+lazy val commonSettings = Jvm.required(Jvm.V7) ++ Seq(
+  organization       := "com.mediative",
+  scalaVersion       := "2.10.5",
+  crossScalaVersions := Seq("2.10.5", "2.11.7"),
```

Comment:
Initially I had both 2.10.4 and 2.10.5 here, but I figured with our dependencies, the version get upgraded to 2.10.5 anyway, so I'm not sure the 2.10.4 build was doing any good and just slow things don't when you want locally to make sure tests work with `+ test`. Travis is still configured with all 3 version though.